src/pybind/mgr/telemetry/tests/telemetry_upgrade_tests.md

```diff
@@ -0,0 +1,213 @@
+The `upgrade` suite is used to verify that upgrades from the (X-2) releases of
+Ceph (where X stands for the current release, i.e tentacle as of writing this
+documentation) to the current stable Ceph release (_tentacle_) can complete
+successfully without disrupting any ongoing workloads.
+
+There are two goals of this suite:
+- Test that the Ceph cluster has been successfully upgraded to the desired
+  version without the workloads (eg: basic rados api tests, import/export cli
+  tests for rbd, stress testing etc) getting disrupted
+- Test Telemetry functions as expected after the upgrade
```

Comment:
```suggestion
```
Let's remove this section since at this stage, you're talking about the upgrade suite as a whole, and this section specifically describes the goals of the telemetry-upgrade sub-suite. Since you talk about these aspects later in the doc, it's better to remove it here.